addons/l10n_es/data/account_fiscal_position_template_data.xml

```diff
@@ -113,6 +113,16 @@
             <field name="chart_template_id" ref="account_chart_template_common"/>
         </record>
 
+        <record id="fp_reagyp_a" model="account.fiscal.position.template">
+            <field name="name">Régimen especial de A.G. y P. - Agricultura</field>
```

Comment:
Call it directly `Régimen especial de agricultura`

Reply:
No, because for AEAT is the same fiscal position, but have two different treatment. All accountant know like "Regimen especual de A. G. y P." or directly "REAGYP". If you suggest a short name will to be "REAGYP - Agricultura"

Reply:
Yeah, it's for shortening the title. Ok what you suggest